platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -168,6 +168,8 @@ struct pktio_entry {
 		} state[PKTIO_MAX_QUEUES];
 		int fd[PKTIO_MAX_QUEUES];
 	} pcapng;
+
+	odp_proto_layer_t parse_layer;
```

Comment:
Moving this to the beginning of the struct (e.g. below `txl`) improves performance a bit as other struct members required in the fast path are located on the first cache line.

Reply:
Oops, I completely forgot to consider the placement. Fixed in v6.